encore/slurm_queue.py

```diff
@@ -14,23 +14,28 @@ def __init__(self, job_id, job_directory, config = None):
         else:
             self.config = dict()
 
-    def get_batch_headers(self, model_plan):
-        mem_per_cpu = model_plan.get("mem_per_cpu", 6500)
-        cores_per_job= model_plan.get("mem_per_cpu", 56)
-
-        return ["#!/bin/bash", 
-           "#SBATCH --partition={}".format(self.config.get("QUEUE_PARTITION", "encore")),
-           "#SBATCH --job-name=gasp_{}".format(self.job_id), 
-           "#SBATCH --mem-per-cpu={}".format(mem_per_cpu), 
-           "#SBATCH --workdir={}".format(self.job_directory), 
-           "#SBATCH --cpus-per-task={}".format(cores_per_job),  
-           "#SBATCH --time=14-0", 
-           "#SBATCH --nodes=1",
-           "export OPENBLAS_NUM_THREADS=1"]
+    def get_batch_headers(self):
+        sbatch_headers = ["#!/bin/bash"]
+
+        if "SLURM_ACCOUNT" in self.config:
+            sbatch_headers.append(
+                "#SBATCH --account={}".format(self.config.get("SLURM_ACCOUNT")))
+
+        sbatch_headers.append(
+            "#SBATCH --partition={}".format(self.config.get("QUEUE_PARTITION", "encore")),
+            "#SBATCH --job-name=gasp_{}".format(self.job_id),
+            "#SBATCH --mem-per-cpu={}".format(self.config.get("JOB_MEM_PER_CPU", 6500)),
```

Comment:
the `--mem-per-cpu`, `--cpus-per-task`, and  `--time` are now driven from flask config options. I prefixed these with `JOB` but I'm not partial to any naming scheme or another. I'd suggest some uniformity for them + `QUEUE_PARTITION` etc though.